content/blog/2021-11-30-november-21-community-gems.md

```diff
@@ -0,0 +1,192 @@
+---
+title: November '21 Community Gems
+date: 2021-11-30
+description: |
```

Comment:
should be `>` to auto-format (in both places)

Reply:
overall - same policy that we applied before - 80 symbols please

Reply:
I'm a little confused on the 80 character limit again. The descriptions are in the same format as the descriptions for previous community gems, which have more than 80 characters. So is this going to be the standard going forward?

Reply:
It was always a standard to wrap lines at 80 symbols.

Using `>` instead of `|` (there are some caveats, but it almost always possible) automates this - linter will format everything for you appropriately. I've updated the most recent Gems (and all blogs, gems were updated a while ago to use `>`, but all of them were formatted at 80 symbols always).